pymc3/tests/test_distributions_random.py

```diff
@@ -539,6 +539,23 @@ def test_probability_vector_shape(self):
         assert pm.Categorical.dist(p=p).random(size=4).shape == (4, 3, 7)
 
 
+class TestDirichlet(SeededTest):
+    def test_dirichlet_random_shape(self):
+        assert pm.Dirichlet.dist(a=np.ones(2)).random(size=(1)).shape == (1, 2)
+        assert pm.Dirichlet.dist(a=np.ones(2)).random(size=(2)).shape == (2, 2)
+        assert pm.Dirichlet.dist(a=np.ones((2, 2))).random(size=(2, 100)).shape == (2, 100, 2, 2)
+        assert pm.Dirichlet.dist(a=np.ones((3, 4))).random(size=(3, 4)).shape == (3, 4, 3, 4)
+        assert pm.Dirichlet.dist(a=np.ones((3, 4))).random(size=(3, 4, 100)).shape == (
+            3,
+            4,
+            100,
+            3,
+            4,
```

Comment:
If you remove this last comma after `4`, the pre-commit will be fine with having everything in one line.

Reply:
Parametrized tests instead.